src/raiden_libs/matrix.py

```diff
@@ -284,6 +284,9 @@ def _handle_message(self, room: Any, event: dict) -> bool:
             return False
 
         for message in messages:
+            if message.sender is None:
```

Comment:
Can that happen? If not, I would prefer an assert.

Reply:
Just checked, it cannot happen. This is checked in `raiden_libs.matrix.deserialize_messages`